benchmarks/benchmarks/bench_function_base.py

```diff
@@ -98,44 +98,122 @@ def time_select_larger(self):
 class Sort(Benchmark):
     params = [
         ['quick', 'merge', 'heap'],
-        ['float32', 'int32', 'uint32']
+        ['float32', 'int32', 'uint32'],
```

Comment:
`float64` would probably be a better check here, and maybe `int64` instead of `uint32`. How does radix sort deal with negative integers, should we test with those?

Reply:
The comment about negative integers applies more to the actual sort PR rather than this benchmark PR.

However, radix sort doesn’t perform any differently on negative integers, so I didn’t bother to include them.

With my optimisations, the range of the numbers does matter. So if all numbers were 0-255, it’s only going to sort by the last byte. Are multiple ranges worth adding?

Reply:
I've added both an unsigned and a signed integer type because they're "different" to radix sort.